runtime/src/main/java/org/corfudb/runtime/collections/PersistedCorfuTable.java

```diff
@@ -5,14 +5,16 @@
 import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
 import org.corfudb.runtime.object.ICorfuSMR;
+import org.corfudb.runtime.object.ICorfuSMRProxyMetadata;
 import org.corfudb.runtime.object.ICorfuSMRProxy;
 import org.corfudb.runtime.object.ICorfuSMRUpcallTarget;
+import org.corfudb.runtime.object.transactions.Transaction;
 
 import javax.annotation.Nonnull;
 import java.util.Map;
-import org.corfudb.runtime.object.ICorfuSMRProxyMetadata;
-
+import java.util.Optional;
 import java.util.Set;
+import java.util.function.Consumer;
```

Comment:
Can we remove the unused imports please? Best is if you can do that cool trick where you optimize all the imports :) thanks